### DISCUSSION TITLE
ARM64 release?

### DISCUSSION THREAD

**Discussion by matthoosier** [audiamus/BookLibConnect#42] [Ideas]
Hi,

Thanks, this program is really useful. Would you consider making an ARM release of the installer?

**Comment by audiamus** [comment]
No, I don't have such plans at the moment. The big task at the moment is to integrate it into AAX Audio Converter.

**Comment by matthoosier** [comment]
Fair enough. I had a go at building this myself (on x86 for now), but after cloning and doing 'Build Solution', I get a bunch of errors like this:

![image](https://user-images.githubusercontent.com/1817136/167257022-93c40864-ffea-4486-9ddd-e2e340af517d.png)

I'm not a very experienced Visual Studio user. Is there some kind of preparatory step I'm should have done to make those dependencies (they look like various other of your projects from the main AaxAudioConvert) available?

**Comment by audiamus** [comment]
The only project dependency that is currently not in the solution is Mbucari's AAXClean. The published version and its associated NuGet package had a bug. But he has fixed this since. So you can swap the AAXClean dependency with the latest NuGet package and the whole thing should build.

**Comment by matthoosier** [comment]
Thanks. I implemented this as PR #43.
